src/cdk-experimental/ui-patterns/radio-group/radio-group.ts

```diff
@@ -41,8 +66,8 @@ export class RadioGroupPattern<V> {
   /** Whether the radio group is readonly. */
   readonly = computed(() => this.selectedItem()?.disabled() || this.inputs.readonly());
 
-  /** The tabindex of the radio group (if using activedescendant). */
-  tabindex = computed(() => this.listBehavior.tabindex());
+  /** The tabindex of the radio group (if using activedescendant or if in toolbar). */
```

Comment:
This comment seems a little funky. I think it's sufficient to not mention the implementation details and just say:

```
/** The tabindex of the radio group. */
```